.github/workflows/ci.linux.x86_64.yml

```diff
@@ -211,6 +211,25 @@ jobs:
           cd build && ctest -E test-lockfree --timeout 3600 -V
           pkill redis-server
 
+  build-from-source:
+    runs-on: ubuntu-latest
+    container:
+      image: almalinux:8
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build
+        run: |
+          dnf -y install git gcc-c++ cmake
+          cmake -B build -D CMAKE_BUILD_TYPE=Debug        \
+                         -D PHOTON_ENABLE_ECOSYSTEM=ON    \
+                         -D PHOTON_BUILD_TESTING=ON       \
+                         -D PHOTON_ENABLE_SASL=ON         \
+                         -D PHOTON_ENABLE_FUSE=ON         \
+                         -D PHOTON_ENABLE_URING=ON        \
+                         -D PHOTON_ENABLE_EXTFS=ON        \
+                         -D PHOTON_BUILD_DEPENDENCIES=ON
```

Comment:
Since we have been using custom image, at least keep one CI to verify `PHOTON_BUILD_DEPENDENCIES`.